src/components/Card.js

```diff
@@ -22,23 +26,32 @@ export default class Card {
 
   _setEventListeners() {
     this._likeButton.addEventListener("click", () => {
-        this._handleLikeIcon();
-      });
+      this._handleLikeIcon(this);
+    });
 
     this._deleteButton.addEventListener("click", () => {
-        this._handleDeleteCard();
+        this._handleDeleteCard(this);
     });
 
     this._cardImageElement.addEventListener('click', () => {
       this._handleImageClick(this._name, this._link);
     });
   }
 
-  _handleLikeIcon() {
-    this._likeButton.classList.toggle(cardselectors.cardLikeToggle);
+  _renderLikeIcon() {
+    if (this._isLiked) {
```

Comment:
`.setLike(cardData.id, cardData.isLiked)`

To make it work you need to change them in class `Card`. 

`this._id` should be `this.id` and `this._isLiked` should be `this.isLiked`